crates/console-timer/src/lib.rs

```diff
@@ -32,7 +32,7 @@ For scenarios where `ConsoleTimer::scope` can't be used, like with
 asynchronous operations, you can use `ConsoleTimer::new` to create a timer.
 The measurement ends when the timer object goes out of scope / is dropped.
 
-```no_run
+```ignore
```

Comment:
What about adding a `dev-dependency`? Exists for precisely this kind of thing ;)

Reply:
Hehe alright, not clear on what best practices are in cases like these. Done! 👍